disco/bot/parser.py

```diff
@@ -139,9 +139,10 @@ def convert(self, ctx, types, value):
             try:
                 return typ(ctx, value)
             except Exception as e:
+                exc = e
```

Comment:
For my sanity, you should declare this in the function scope and just set to None. Will provide a slightly more clear exception in the case we break through.